contents/blog/ab-testing-mistakes.md

```diff
@@ -0,0 +1,175 @@
+---
+date: 2023-04-27
+title: "8 A/B testing mistakes and how to avoid them"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ['lior-neu-ner']
+featuredImage: ../images/blog/experiments.jpeg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+---
+
+Making a mistake in an A/B test can have disastrous implications. At best, it means you've wasted weeks waiting for results that are not useful. At worst, it can lead you to make the wrong product decisions. This blog post will explore five common A/B testing mistakes to avoid to ensure successful experiments!
+
+## 1. Including unaffected users in your experiment
+
+Another common mistake in A/B testing is counting users in your experiment who aren't actually affected by the change you're testing. This can happen if you're not careful in how and when you assign users to the control and test groups. It dilutes your experiment results, making it harder to determine the impact of your changes. 
```

Comment:
```suggestion
The first common mistake in A/B testing is counting users in your experiment who aren't actually affected by the change you're testing. This can happen if you're not careful in how and when you assign users to the control and test groups. It dilutes your experiment results, making it harder to determine the impact of your changes. 
```